src/components/GoalsComponents/ShareGoalModal/ShareGoalModal.tsx

```diff
@@ -179,18 +179,13 @@ const ShareGoalModal = ({ goal }: { goal: GoalItem }) => {
           </div>
           {displaySubmenu === "contacts" && (
             <div className="shareWithContacts">
-              {contacts.length === minContacts && (
+              {contacts.length === 0 && (
```

Comment:
@siva9495 this will only remove the text warning. but will not remove the limit of contacts u can add. @tijlleenders can you confirm here whether you want to remove the limit completely or just want to remove the warning

Reply:
@Tushar-4781 I think we agreed on a limit of 10 contacts.

Reply:
i think it was 3. never mind. @siva9495 there is a variable at the start of function code which is set to 3 just change it to 10 and then we are gtg

Reply:
ok @Tushar-4781.